inference-engine/src/mkldnn_plugin/mkldnn_graph.h

```diff
@@ -47,6 +47,7 @@ class MKLDNNGraph {
     void setConfig(const Config &cfg);
     void setProperty(const std::map<std::string, std::string> &properties);
     Config getProperty();
+    Config getProperty() const;
```

Comment:
this method is rdundant (as original function already returns the copy of the Config, just make the original func const.
otherwise, the PR is LGTM